test/e2e/config/ibmcloud-e2e.yaml

```diff
@@ -0,0 +1,58 @@
+managementClusterName: capi-ibmcloud-e2e
+
+providers:
+  - name: cluster-api
+    type: CoreProvider
+    versions:
+    - name: v1.0.2
+      value: https://github.com/kubernetes-sigs/cluster-api/releases/download/v1.0.2/core-components.yaml
+      type: url
+      files:
+      - sourcePath: "../data/shared/metadata.yaml"
+  - name: kubeadm
+    type: BootstrapProvider
+    versions:
+    - name: v1.0.2
+      value: https://github.com/kubernetes-sigs/cluster-api/releases/download/v1.0.2/bootstrap-components.yaml
+      type: url
+      files:
+      - sourcePath: "../data/shared/metadata.yaml"
+  - name: kubeadm
+    type: ControlPlaneProvider
+    versions:
+    - name: v1.0.2
+      value: https://github.com/kubernetes-sigs/cluster-api/releases/download/v1.0.2/control-plane-components.yaml
+      type: url
+      files:
+      - sourcePath: "../data/shared/metadata.yaml"
+  - name: ibmcloud
+    type: InfrastructureProvider
+    versions:
+    - name: v0.2.0
+      value: "${PWD}/config/default"
+      files:
+      - sourcePath: "${PWD}/metadata.yaml"
```

Comment:
How does this `${PWD}` when running the tests from a different folder than root_dir of a project?

Reply:
Ideally the tests would be run from the root directory. In cases of running the tests from a different folder, this wouldn't work since the `${PWD`} would be resolved on the current path.

Reply:
then can `../../../metadata.yaml` resolved this issue? if not then we can stick with `${PWD}`

Reply:
For consistency, I added `${PWD}` for all